security/pfSense-pkg-snort/files/usr/local/www/snort/snort_interfaces_edit.php

```diff
@@ -600,7 +597,7 @@ function snort_get_config_lists($lists) {
 }
 
 // If using Inline IPS, check that CSO, TSO and LRO are all disabled
-if ($pconfig['enable'] == 'on' && $pconfig['ips_mode'] == 'ips_mode_inline' && (!isset($config['system']['disablechecksumoffloading']) || !isset($config['system']['disablesegmentationoffloading']) || !isset($config['system']['disablelargereceiveoffloading']))) {
+if ($pconfig['enable'] == 'on' && $pconfig['ips_mode'] == 'ips_mode_inline' && (!config_path_enabled('system', 'disablechecksumoffloading') || !config_path_enabled('system', 'disablesegmentationoffloading') || !config_path_enabled('system', 'disablelargereceiveoffloading'))) {
```

Comment:
No need to do this now, but at some point in the future it would be nice to break up long lines like this so each test is on a separate line inside the `if()` statement so it's easier to read. Also each test should be in its own set of parenthesis.

```php
if (a &&
    b &&
    c) {
     do_stuff();
}
```

Reply:
While I was in the code, went ahead and made the modification for easier reading. Probably a few other instances elsewhere that can be addressed in future updates.